letsencrypt/client.py

```diff
@@ -189,7 +189,7 @@ def __init__(self, config, account_, auth, installer, acme=None):
             self.auth_handler = None
 
     def obtain_certificate_from_csr(self, domains, csr,
-        typ=OpenSSL.crypto.FILETYPE_ASN1):
+        typ=OpenSSL.crypto.FILETYPE_ASN1, authzr=False):
```

Comment:
We should use `authzr=None` here. Additionally, this option should be documented in the docstring.